data/hails.txt

```diff
@@ -465,10 +483,12 @@ phrase "friendly civilian"
 		"the best of luck"
 		"fair winds and bright skies"
 		"Allah"
+		"Huitzilopochtli"
 		"the power of Heaven"
 		"Heaven's blessings"
 		"the Way and the Light"
 		"the immortal Tao"
+		"the great and powerful kek"
```

Comment:
Praise Kek.
Show respect to your god and capitalize Kek.

Reply:
As long as we're throwing memes into this, do you think you could add hails referring to the [Helix Fossil](https://s2.qwant.com/thumbr/0x0/e/1/1c8ef2030e45eff43d3597bc5adc42/b_1_q_0_p_0.jpg?u=https%3A%2F%2Fih1.redbubble.net%2Fimage.23302549.0433%2Fflat%2C1000x1000%2C075%2Cf.jpg&q=0&b=1&p=0&a=1) and [Magic Conch](https://s1.qwant.com/thumbr/0x0/a/9/1c8bf428d63fd3cc54c5079583e46b/b_1_q_0_p_0.jpg?u=http%3A%2F%2Fi1.ytimg.com%2Fvi%2FLQyDNs8ZzHI%2Fmaxresdefault.jpg&q=0&b=1&p=0&a=1)? It relates to something I have planned for #2908 and making it canon through hails would be a plus.

Reply:
What kind of reference should I make? Just anything? I could find one of them and mention some scientists who found a Helix fossil buried somewhere, but idk how I'd implement the magic conch, ha.

Reply:
>What kind of reference should I make? 

References as if they were religions, so just under this Kek hail.